html/index.html.in

```diff
@@ -113,20 +113,23 @@
                             <option></option>
                         </select>
                     </div>
-                    <ul class="nav navbar-nav navbar-right navbar-collapse collapse">
-                        <li>
-                            <a href="#translation" data-mode="translation" data-text="Translation" class="hide" data-toggle="tab">Translation</a>
-                        </li>
-                        <li>
-                            <a href="#analyzation" data-mode="analyzation" data-text="Morphological_Analysis" class="hide" data-toggle="tab">Morphological analysis</a>
-                        </li>
-                        <li>
-                            <a href="#generation" data-mode="generation" data-text="Morphological_Generation" class="hide" data-toggle="tab">Morphological generation</a>
-                        </li>
-                        <li>
-                            <a href="#sandbox" data-mode="sandbox" data-text="APy_Sandbox" class="hide" data-toggle="tab">APy sandbox</a>
-                        </li>
-                    </ul>
+                    <div class="navbarLinks">
```

Comment:
Is there a reason we can't just add this class to the `ul` itself?

Reply:
Actually `div` provides us a better way to structure our content in html. This is the reason I preferred adding the class to `div` rather than class `ul` itself.

Reply:
I'd like to reduce the layering of this HTML unless necessary. If the `ul` can take the class and retain the desired functionality, please add it there. A `ul` container is sufficiently semantic when the `div` would only have one child.

Reply:
I have added the class to `ul` instead of `div` and it seems to work just fine to me.